govc/vm/disk/create.go

```diff
@@ -61,6 +62,7 @@ func (cmd *create) Register(ctx context.Context, f *flag.FlagSet) {
 	f.Var(&cmd.Bytes, "size", "Size of new disk")
 	f.BoolVar(&cmd.Thick, "thick", false, "Thick provision new disk")
 	f.BoolVar(&cmd.Eager, "eager", false, "Eagerly scrub new disk")
+	f.StringVar(&cmd.DiskMode, "mode", "dependent", "One of: dependent (also known as persistent), persistent, nonpersistent, undoable, independent_persistent, independent_nonpersistent, append")
```

Comment:
Sure. With the pipe in between on the usage output, or specifically joining them from an array of options for the usage?